Team_Roster/pages/depthChart.py

```diff
@@ -1,44 +1,99 @@
-from flask import render_template
+from flask import jsonify, render_template, request
 from app import db
-from app.models import People, Fielding, Team
+from app.models import People, Fielding, Batting
 from sqlalchemy import func
 
-def ShowDepthChart(teamId, year):
-    #Query to get player fielding time in each position for the specified team and year
-    subquery = (
-        db.session.query(
-            Fielding.position,
-            Fielding.playerID,
-            func.sum(Fielding.f_InnOuts).label("total_innouts"),
+def ShowDepthChart(teamId):
```

Comment:
are you fetching all the stats, or just the one displayed?

Reply:
I'm not quite sure what you mean, but currently only the ones actually displayed are being retrieved at the moment.